experiment_scheduler/master/master.py

```diff
@@ -292,6 +298,40 @@ def _wrap_by_task_status(self, task_id, status):
             status=status,
         )
 
+    def edit_task(self, request, context):
+        task_id = request.task_id
+        cmd = request.cmd
+        if task_id in self.queued_tasks.keys():
+            self.queued_tasks[task_id].command = cmd
+            response = MasterResponse(
+                experiment_id="0", response=MasterResponse.ResponseStatus.SUCCESS
```

Comment:
Currently MasterResponse returns "0" since edit response does not need experiment id.